relayer/src/chain/cosmos.rs

```diff
@@ -440,6 +440,11 @@ impl CosmosSdkChain {
         self.config.max_gas.unwrap_or(DEFAULT_MAX_GAS)
     }
 
+    /// Get the fee granter address
+    fn fee_grant(&self) -> &str {
```

Comment:
Could we name this method similarly to the config.toml option, specifically `fee_granter` ?